core/src/main/java/org/nervos/ckb/response/item/Seal.java

```diff
@@ -0,0 +1,20 @@
+package org.nervos.ckb.response.item;
+
+import java.util.List;
+
+/**
+ * Created by duanyytop on 2018-12-21.
+ * <p>
+ * Copyright © 2018 Nervos Foundation. All rights reserved.
+ */
+public class Seal {
+
+    /**
+     * nonce : 10545468520399447721
+     * proof : [163,13,0,0,12,17,0,0,98,28,0,0,240,60,0,0,200,62,0,0,12,76,0,0,6,93,0,0,247,93,0,0,107,97,0,0,230,100,0,0,16,103,0,0,244,107,0,0]
+     */
+
+    public String nonce;
+    public List<Integer> proof;
```

Comment:
This should be list of `uint8`.

Reply:
It’s indeed a block of bytes.

Reply:
`byte[]`

Reply:
If we use byte as uint8 as follow :

```java
@Test
    public void testSeal() {
        String json = "{\"nonce\": 10545468520399447721, \"proof\": [128, 127]}";
        Seal seal = new Gson().fromJson(json, Seal.class);

        Assert.assertEquals("10545468520399447721", seal.nonce);
        Assert.assertEquals(seal.proof, Arrays.asList(128, 127));
    }

    public static class Seal {
        public String nonce;

        // data type: uint8
        private List<Byte> proof;
    }
```
And the result is :

```java
java.lang.AssertionError: 
Expected :[-128, 127]
Actual   :[128, 127]
```